portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/rolesandpermissions/cproles/RolesAdmin.testcase

```diff
@@ -108,6 +108,83 @@ definition {
 			value1 = "No roles were found.");
 	}
 
+	@description = "This test automates LPS-141903 and asserts that users are not counted in Roles after deactivating them"
+	@priority = "3"
+	test DeactivatedUsersAreNotCountedInRoles {
```

Comment:
This is a test only didactive one user actually, and it's a test for site role, let's rename the test and update the description.